### DISCUSSION TITLE
p/49fb661c297b4544a208ea898c77e5a0/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#31] [Announcements]
# p/49fb661c297b4544a208ea898c77e5a0/

场景vite 作为现代 web 应用的构建工具，许多开发者将之用于开发 web 应用（react、vue），由于其本身的易用性和高性能，许多 web 框架甚至官方编写了插件（solid、astro），可以称得上是近年来 webpack 的成功的挑战者。但实际上，发展至今，vite 已经远不仅仅是一个 web 层的工具，它的周边生态正在蓬勃发展，衍生除了一系列周边的工具，在之前的 vite-不仅仅是

https://blog.rxliuli.com/p/49fb661c297b4544a208ea898c77e5a0/

**Comment by miaopass-future** [comment]
举例 ：
1、1+1=2
2、2+2=4
3、好了，现在我们了解了数字计算方法，下面开始求以下列函数极限、求微积分、求曲线围成的图形旋转后的旋转体体积

**Reply by rxliuli** [reply]
vite 插件编写和 sourcemap 处理可能确实需要一些背景知识 emmm

**Comment by oneyoung19** [comment]
写得非常好，答疑解惑。

**Reply by rxliuli** [reply]
很高兴能帮到你 😌
